src/test/java/core/basesyntax/operation/BalaceOperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalaceOperationHandlerTest {
+    private static final FruitTransaction fruitTransaction
+            = new FruitTransaction("b", new Fruit("banana"), 10);
+    private static final Fruit fruit = new Fruit("banana");
+    private static BalaceOperationHandler balaceOperationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        balaceOperationHandler = new BalaceOperationHandler();
+    }
+
+    @Test
+    public void apply_validFruitTransaction_Ok() {
+        balaceOperationHandler.apply(fruitTransaction);
+        Integer expected = 10;
+        Integer actual = Storage.storage.get(fruit);
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void apply_NullFruitTransaction_NotOk() {
```

Comment:
please check naming of all test methods in all classes
and let's bring them to the same pattern